gocourse05/server.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+
+	"gocourse05/camera"
+)
+
+const serverUrl = "http://remote.animals.control"
+
+type Memory []camera.ProcessedData
+
+func saveProcessedData(p camera.Processor, m *Memory) error {
+	processedData, err := p.ProcessData()
+	if err != nil {
+		return err
+	}
+
+	*m = append(*m, *processedData)
+
+	return nil
+}
+
+func sendProcessedData(m Memory, url string) error {
+	dataToSend := ""
+	for _, data := range m {
+		dataToSend = dataToSend + data.AnimalMovement
+	}
+
+	if len(dataToSend) > 0 {
+		fmt.Printf("Sending data \"%s\" to %s", dataToSend, url)
```

Comment:
Можна ось так: 
```suggestion
		fmt.Printf("Sending data %q to %s", dataToSend, url)
```

згідно https://pkg.go.dev/fmt

> ```
> %q	a single-quoted character literal safely escaped with Go syntax.
> ```